web/src/components/Toolbar.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <v-app-bar
+    app
+    color="primary"
+    dark
+    src="../assets/background.png"
+  >
+    <v-btn
+      href="/"
+      text
+    >
+      <v-toolbar-title>
+        SpeakHer
+      </v-toolbar-title>
+    </v-btn>
+
+    <v-spacer />
+
+    <v-toolbar-items class="d-flex align-center">
```

Comment:
because there is only one attribute. the eslint vue/preferred will do it for us. We could also set up husky hooks or GH actions to do these checks for us as well 😄